src/Phinx/Db/Adapter/DryRunAdapter.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Phinx
+ *
+ * (The MIT license)
+ * Copyright (c) 2015 Rob Morgan
```

Comment:
Far as I can tell, this is what's done in the files you've contributed, so I just followed suit.